platform/broadcom/sai-modules.mk

```diff
@@ -5,4 +5,15 @@ BRCM_OPENNSL_KERNEL_VERSION = 4.3.0.10-2
 BRCM_OPENNSL_KERNEL = opennsl-modules_$(BRCM_OPENNSL_KERNEL_VERSION)_amd64.deb
 $(BRCM_OPENNSL_KERNEL)_SRC_PATH = $(PLATFORM_PATH)/saibcm-modules
 $(BRCM_OPENNSL_KERNEL)_DEPENDS += $(LINUX_HEADERS) $(LINUX_HEADERS_COMMON)
+$(BRCM_OPENNSL_KERNEL)_MACHINE = broadcom
 SONIC_DPKG_DEBS += $(BRCM_OPENNSL_KERNEL)
+
+# SAI bcm modules for DNX family ASIC
```

Comment:
Having different set of modules for DNX vs XGS will preclude any board designs that contain both types of chips.  The linux-kernel-bde is a simple device enumerator.  I think it can easily be made to support both types of chips.

Reply:
Currently the SDK version is different for DNX ( it uses 6.5.22 ) and XGS ( it uses 6.5.21) + the bcm knet modules DNX family needs special flags for the build. So both the SAI packages and OPENNSL modules packages needs to be separate for each of the ASIC families.

We have later look into with broadcom if there is a use case where both chips are used in the same board. Do you foresee such a board use case ?